src/main/java/org/skriptlang/skript/bukkit/spawner/elements/expressions/spawnerentry/equipment/chances/ExprEquipmentDropsWithChance.java

```diff
@@ -0,0 +1,61 @@
+package org.skriptlang.skript.bukkit.spawner.elements.expressions.spawnerentry.equipment.chances;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.spawner.SpawnerModule;
+import org.skriptlang.skript.bukkit.spawner.util.SpawnerEntryEquipment.Drops;
+
+@Name("Spawner Entry - Equipment Drop with Chance")
+@Description("Returns the drop chance of an already existing equipment drop.")
+@Examples("set {_chance} to drop chance of {_equipment drop chance}")
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.21+")
+public class ExprEquipmentDropsWithChance extends SimplePropertyExpression<Drops, Float> {
+
+	static {
+		registerDefault(SpawnerModule.SYNTAX_REGISTRY, ExprEquipmentDropsWithChance.class, Float.class,
+				"drop chance", "equipmentdropchances");
```

Comment:
same as before, should include `spawner`, be more precise as this could create many conflicts.